### DISCUSSION TITLE
Welcome to MPC-LiveXplore Discussions!

### DISCUSSION THREAD

**Discussion by TheKikGen** [TheKikGen/MPC-LiveXplore#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by philippeback** [comment]
Hello, making beats on MPC X, Live2. Interested in getting other MIDI controllers to work with the MPC for the global functions.
Basically having DJTT TF working on a Live2 and why not on the X.

Custom arp patterns also useful.

Generally curious about how things do work.

Having serious linux experience + can hold a soldering iron and then some.

Currently wondering how I can get ssh to work on a 2.12 update.

**Comment by RuralGalaxy** [comment]
Hi, i'm trying to change splash screen, how can I recover de original splash screen to convert with imagemagick to png to modify and put inside MPC again?

**Comment by serversonic** [comment]
With regard to MPC contacting Google Analytics, licence checking seems to break if it's blocked, specifically with a PiHole, but if a Live 2 isn't connected at all, licensed plugins continue to work.

**Comment by twiddler1111** [comment]
Is there a way to add a cursor and controll the mpc live 2 with a mouse?
Other thing on my wishlist would be more complex lfos and maybe more lfos in general.
